snforge_std/src/trace.cairo

```diff
@@ -2,48 +2,55 @@ use core::result::ResultTrait;
 use core::starknet::testing::cheatcode;
 use core::starknet::ContractAddress;
 
+/// Tree-like structure which contains all of the calls and sub-calls along with the results
 #[derive(Drop, Serde, PartialEq)]
 struct CallTrace {
-    entry_point: CallEntryPoint,
-    nested_calls: Array<CallTrace>,
+    entry_point: CallEntryPoint, /// The topmost entry point of this trace
```

Comment:
Wouldn't it be better to put it above the field definition?

Reply:
done